core/file/atime_spec.rb

```diff
@@ -4,15 +4,23 @@
   before :each do
     @file = tmp('test.txt')
     touch @file
+    3.times do
+      @atime = File.atime(@file)
+      break if @atime.usec > 0
+      sleep 0.1
+    end
```

Comment:
Please move this directly in the concerned spec as other specs do not need this, and the sleep wastes time.